tests/tools/nnpackage_run/src/rawformatter.cc

```diff
@@ -30,13 +30,10 @@ void RawFormatter::loadInputs(const std::string &filename, std::vector<Allocatio
   NNPR_ENSURE_STATUS(nnfw_input_size(session_, &num_inputs));
 
   // TODO: Support multiple inputs
-  // Option 1. Get comman-separated input file list like --load:raw in.0,in.1,in.2
-  // Option 2. Get prefix --load:raw out
-  //           Internally access out.0, out.1, out.2, ... out.{N} where N is determined by api.
-  if (num_inputs != 1)
-  {
-    throw std::runtime_error("Only 1 input is supported for raw input");
-  }
+  // Option 1: Get comman-separated input file list like --load:raw a,b,c
+  // Option 2: Get prefix --load:raw in
+  //           Internally access in.0, in.1, in.2, ... in.{N-1} where N is determined by nnfw info
+  //           query api.
```

Comment:
I thought one more option (= Option3) before this PR.
```
Option 3: Get template --load:raw in.#.raw
                Get in.0.raw, in.1.raw, ... in.{N-1}.raw.
```
But I am not sure it is better to use for user. Also, it requires more implementation.

@hseok-oh Do you like this option? Or do you have any idea or suggestion?

Reply:
No. Let's update package's raw data file name.